CMakeLists.txt

```diff
@@ -99,6 +99,7 @@ add_executable(queue_IO_check test/queue_IO_check.cxx)
 target_link_libraries(queue_IO_check pthread ${Boost_PROGRAM_OPTIONS_LIBRARY} )
 
 
+add_library(appfwk_DummyModule_duneDAQModule test/DummyModule.cpp)
```

Comment:
we don't need a new test module to implement this idea

Reply:
It can be removed, I don't mind. But it may come back.
A do-nothing dumb-as-a-rock module tends to be useful for very basic unit tests.